Project/Packages/manifest.json

```diff
@@ -1,18 +1,23 @@
 {
   "dependencies": {
+    "com.unity.ide.rider": "3.0.5",
+    "com.unity.inputsystem": "1.1.0-preview.3",
     "com.unity.ml-agents": "file:../../com.unity.ml-agents",
     "com.unity.ml-agents.extensions": "file:../../com.unity.ml-agents.extensions",
     "com.unity.nuget.newtonsoft-json": "2.0.0",
     "com.unity.test-framework": "1.1.22",
+    "com.unity.toolchain.macos-x86_64-linux-x86_64": "0.1.20-preview",
```

Comment:
Should this (and the rider package) be here?

Reply:
this was added automatically.  I added rider back because it makes quite a difference in development experience.